passwordreset.html

```diff
@@ -1,30 +1,47 @@
 <html>
-    <head>
-        <link rel="stylesheet" type="text/css" href="style.css" /> 
-        <title>EMS- Password Reset</title>
-        <script src="https://cdn.tailwindcss.com"></script>
-    </head>
-    <body>
-    <div class="nav">
-    EMPLOYEE MANAGEMENT SYSTEM    
-    </div>
-   
-    <div class="title">Forget Password</div>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>EMS- Password Reset</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <style type="text/tailwindcss">
+      @layer utilities {
+  a {
+    color: rgb(193, 255, 255);
+  }
+  body {
+    background-color: rgb(177, 185, 192);
+  }
+  label {
+    color: rgb(180, 180, 180);
+    font-size: large;
+  }
+  input {
+    border: none;
+    padding: 4px 10px;
+    width: 300px;
+    font-size: large;
+  }
+  .error {
+    color: rgb(250, 134, 134);
+  }
+}
+    </style>
```

Comment:
Pls make a separate CSS for this page as well.

Reply:
fixed